src/clp_bench/elasticsearchunstructured_executor.py

```diff
@@ -78,34 +83,35 @@ def ingest(self, mode: BenchmarkingMode):
             ratio_match = re.search(r"Compression ratio for \S+ is (\d+\.\d+)", output)
             ingest_e2e_match = re.search(r"Ingestion time for \S+ is (\d+\.\d+) s", output)
             if decompressed_size_match:
-                self.benchmarking_reseults[mode].decompressed_size = (
-                    f"{int(decompressed_size_match.group(1)) / 1024 / 1024}MB"
+                self.benchmarking_results[mode].decompressed_size = (
+                    BenchmarkingResult.get_mb_from_byte(int(decompressed_size_match.group(1)))
                 )
                 logger.info(
-                    f"File size before compression: {self.benchmarking_reseults[mode].decompressed_size}"
+                    "File size before compression: "
+                    f"{self.benchmarking_results[mode].decompressed_size}"
                 )
             else:
                 logger.error("Cannot get decompressed metric")
             if compressed_size_match:
-                self.benchmarking_reseults[mode].compressed_size = (
-                    f"{int(compressed_size_match.group(1)) / 1024 / 1024}MB"
+                self.benchmarking_results[mode].compressed_size = (
+                    BenchmarkingResult.get_mb_from_byte(int(compressed_size_match.group(1)))
                 )
                 logger.info(
-                    f"File size after compression: {self.benchmarking_reseults[mode].compressed_size}"
+                    f"File size after compression: "
+                    f"{self.benchmarking_results[mode].compressed_size}"
                 )
             else:
                 logger.error("Cannot get compressed metric")
             if ratio_match:
-                self.benchmarking_reseults[mode].ratio = f"{ratio_match.group(1)}x"
-                logger.info(f"Compression ratio: {self.benchmarking_reseults[mode].ratio}")
+                self.benchmarking_results[mode].ratio = f"{ratio_match.group(1)}x"
+                logger.info(f"Compression ratio: {self.benchmarking_results[mode].ratio}")
             else:
                 logger.error("Cannot get compression ratio metric")
             if ingest_e2e_match:
-                self.benchmarking_reseults[mode].ingest_e2e_latency = (
-                    f"{ingest_e2e_match.group(1)}s"
-                )
+                self.benchmarking_results[mode].ingest_e2e_latency = f"{ingest_e2e_match.group(1)}s"
                 logger.info(
-                    f"Elasticsearch compressed data in {dataset_path} successfully in {ingest_e2e_match.group(1)} seconds"
+                    f"Elasticsearch compressed data in {dataset_path} successfully in "
+                    f"{ingest_e2e_match.group(1)} seconds"
```

Comment:
This is not an issue, they are just some middle log information, and anyway we will print the final results. So don't care much about this. Also I separate it from one line to to line is because it exceeds 100.